src/service/core-services.js

```diff
@@ -26,6 +26,8 @@ import {installGlobalSubmitListenerForDoc} from '../document-submit';
 import {installHiddenObserverForDoc} from './hidden-observer-impl';
 import {installHistoryServiceForDoc} from './history-impl';
 import {installImg} from '../../builtins/amp-img';
+import {installInaboxCidService} from '../inabox/inabox-cid';
+import {installInaboxViewportService} from '../inabox/inabox-viewport';
```

Comment:
Are these pure imports? I'm worried this will bring in a bunch of inabox files into the v0.js runtime.

Reply:
Moved the method to amp-inabox.js so we don't rely on DCE. And added a test to make sure same services are installed.